fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/InsertUtils.java

```diff
@@ -162,6 +162,10 @@ private static InternalService.PDataRow getRowStringValue(List<NamedExpression>
     private static void beginBatchInsertTransaction(ConnectContext ctx,
             String dbName, String tblName, List<Column> columns) {
         TransactionEntry txnEntry = ctx.getTxnEntry();
+        if (txnEntry.isTransactionBegan()) {
+            throw new AnalysisException(
+                    "Transaction insert can not insert into values and insert into select at the same time");
```

Comment:
why `isTransactionBegan` means use insert into values with insert into select at same time?

Reply:
`isTransactionBegan` meas the `insert into select` already began a txn.
This function will begin a txn for `insert into values`. 
Currently we forbid it, maybe support it later.